Gemfile

```diff
@@ -10,6 +10,7 @@ end
 gem "dotenv-rails"
 gem "rails", "5.2.4.5"
 
+gem "active_hash", "~> 2.3.0"
```

Comment:
I used this version to match https://github.com/simpledotorg/simple-server/pull/2131/files
So we shouldn't get any conflicts.